### PR TITLE
Updated hello-blockstack docs to match latest

### PR DESCRIPTION
**Pull request by tomrice**
Light updates to the `hello-blockstack` tutorial to match the current developer experience.

### CODE REVIEW

**Comment by moxiegirl** [comment]
@tomrice Thank you for the PR. I'm going to pause this review...looks like you want to switch the examples over to Typescript.  We actually need to discuss this as a team.  I'll restart (and adjust my review) after we decide if we want to do this.

**Comment by tomrice** [comment]
@moxiegirl Appreciate your consideration, and I understand the need for due diligence. My intent with this PR is to help the tutorial docs match the live code that's currently generated by yeoman.

The `app.js` code was not rewritten by me. It was recently updated in https://github.com/blockstack/blockstack-app-generator/commit/3eeb20bcc67d70bc804001bc5abb28f600bee2f6. Presently, the live `hello-blockstack` tutorial docs reference `app.js`as it was prior to @thinkverse's commit.

**Comment by moxiegirl** [comment]
@tomrice I actually tested the whole tutorial --- each command --- yesterday while I was commenting in a clean directory. The code was not coming out that way in my test env. So, looks like there is an issue somewhere in the packaging that was getting installed. I'll look into that and hopefully pick this back up.

File: _browser/hello-blockstack.md [hunk 1]
```diff
@@ -211,31 +212,34 @@ All of the code in the file is wrapped in an event
 listener.
 
 ```js
-document.addEventListener("DOMContentLoaded", function(event) {
```
[the remark this answers is not in the document]
Comment (moxiegirl) [reply _browser/hello-blockstack.md]: My search was bad.  I see this now.

**Pull request merged**